newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -748,6 +814,20 @@ export default class AuthenticatedUserProvider extends React.Component<
                 error={this.state.authError}
               />
             )}
+            {this.state.additionalUserInfoDialogOpen && (
+              <AdditionalUserInfoDialog
+                onClose={() => {
+                  this.openAdditionalUserInfoDialog(false);
+                  this.showUserSnackbar({
+                    message: <Trans>👋 Welcome to GDevelop!</Trans>,
```

Comment:
Should it try to show the user username as for after the additional info dialog is closed?